GoTwitterCloneBackend/GoTwitterCloneApp/controller/followhandlers/stopfollowhandler.go

```diff
@@ -29,15 +30,6 @@ func StopFollowHandler(w http.ResponseWriter, r *http.Request) {
 		FollowerUsername: requesterUsername,
 	}
 
-	err = followservice.StopFollowing(r.Context(), stopFollowRequest)
-	if err != nil {
-		responseHelper.SetJsonResponse(http.StatusBadRequest, []string{
-			"failed_unfollowing",
-		})
-		return
-	}
-
+	events.PublishEventMessage(r.Context(), eventhandlers.UnfollowEventHandlerName, stopFollowRequest)
```

Comment:
also added trigger unfollow event handler to separate execution from http request. should reduce latency